ethcore/types/src/receipt.rs

```diff
@@ -129,6 +129,10 @@ pub struct RichReceipt {
 	pub log_bloom: Bloom,
 	/// Transaction outcome.
 	pub outcome: TransactionOutcome,
+	/// Receiver address
+	pub to: Option<H160>,
```

Comment:
This is an `Option` because it missing when it is `Action::Create` transaction

Reply:
Might be good to add this as regular comment.